docs/user/contextplugin.md

```diff
@@ -0,0 +1,27 @@
+# FlowForge Persistent Context Plugin
+
+Some Node-RED flows need a way to persist context values between restarts and FlowForge stack 
+updates but context data is ephemeral by default. With FlowForge Premium, persistent context 
+storage is possible and it survives restarts, upgrades and more.
+
+## Usage
+
+In your Node-RED, you will now have the option to store values in 2 context stores...
+* **Memory**: This is for volatile context where you do not want it to be persisted
+* **Persistent**: This is for persistent context where you want values to persist restarts and upgrades
+
+For more information on Context, head over to the [Node-RED User Guide](https://nodered.org/docs/user-guide/)
+
+## Deployment Considerations
+
+When a project is deployed to a device, Node-RED will use the devices memory for the **Memory** store
+and the devices filesystem for the **Persistent** store
+
+## Limitations
+
+Currently, only async versions of context `get` and context `set` functions are supported. 
```

Comment:
What does that mean? `async`? What does the user need to know? What are the `get` and `set` functions? Which are missing, and how does that influence my day to day?

Reply:
Because without this information they will not know to avoid sync version of these functions and will cause issues in flows.